src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/AttachmentMapper.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.companieshouse.api.strikeoffobjections.controller;
+
+import org.mapstruct.Mapper;
+import org.springframework.stereotype.Component;
+
+import uk.gov.companieshouse.api.strikeoffobjections.model.entity.Attachment;
+import uk.gov.companieshouse.api.strikeoffobjections.model.response.AttachmentResponseDTO;
+
+@Component
+@Mapper(componentModel = "spring")
+public interface AttachmentMapper {
+
+    AttachmentResponseDTO attachmentEntityToAttachmentResponseDTO(Attachment attachment);
```

Comment:
In a nutshell, that's the mapper implementation. All the getting/setting of attributes is done by mapstruct's auto-generated code.